subsys/shell/modules/kernel_service.c

```diff
@@ -107,7 +107,7 @@ static void shell_stack_dump(const struct k_thread *thread, void *user_data)
 
 	shell_print((const struct shell *)user_data,
 		"0x%08X %-10s (real size %u):\tunused %u\tusage %u / %u (%u %%)",
```

Comment:
Should you also fix the formatter to handle the a 64-bit pointer?  would this be better as `%p`?

Reply:
In most of the cases you are only interested in the lower 32bits but it doesn't harm I guess.